tests/integration/test_app.py

```diff
@@ -397,7 +398,7 @@ async def _get_mongodb_uri_from_secrets(ops_test, model: Model) -> str | None:
 
     juju_secrets = await model.list_secrets()
     for secret in juju_secrets["results"]:
-        if secret.label == "database.2.user.secret":
+        if re.match(r"database\.(\d+)\.user\.secret", secret.label):
```

Comment:
It seems that a relation has been added to mongodb-k8s. The secret is now `database.3.user.secret`.